static/src/javascripts/projects/common/modules/commercial/create-ad-slot.js

```diff
@@ -94,13 +94,16 @@ define([
         }
     };
 
-    if (config.switches.mobileTopBannerRemove && $('.top-banner-ad-container--ab-mobile').length > 0 && detect.getBreakpoint() === 'mobile') {
+    if (!!config.tests.mobileTopBannerRemove && detect.getBreakpoint() === 'mobile') {
```

Comment:
It is casting to a boolean. Will make non-boolean to a inverted boolean, then inverts it to get the original value as boolean. Basically this, shorter Boolean(variable) === !!variable